modules/ROOT/pages/hl7-edi.adoc

```diff
@@ -524,13 +518,13 @@ The HL7 message contains the following keys:
 [%header,cols="40a,60a"]
 |===
 |Key name |Description
-|ACK (read only) |ACK message generated in response to the input data. The MSA-1 acknowledgment code value is based on the parser configuration settings. To send an acknowledgment, see the Sending Acknowledgments section below.
-|Data (read or write) |Wrapper for message data, with a key matching the message structure ID value linking to the actual data. This allows different messages to be included in the metadata and handled in DataWeave mappings.
-|Delimiters (read only) |The delimiters used for the message. The characters in the string are interpreted based on position, in the following order: (component separator), (repetition separator), (escape character), (subcomponent separator).
-|Errors (read only) |A list of errors which are associated with the input message. See the HL7Error structure description in the Reading and Validating HL7 Messages section below.
-|Id |Message structure ID.
-|MSH (read only) |Link to received MSH segment data.
-|Name (read only) |Message structure name.
+|`ACK` (read only) |ACK message generated in response to the input data. The MSA-1 acknowledgment code value is based on the parser configuration settings. To send an acknowledgment, see the Sending Acknowledgments section below.
+|`Data` (read or write) |Wrapper for message data, with a key matching the message structure ID value linking to the actual data. This allows different messages to be included in the metadata and handled in DataWeave mappings.
+|`Delimiters` (read only) |The delimiters used for the message. The characters in the string are interpreted based on position, in the following order: (component separator), (repetition separator), (escape character), (subcomponent separator).
+|`Errors` (read only) |A list of errors which are associated with the input message. See the HL7Error structure description in the Reading and Validating HL7 Messages section below.
+|`Id` |Message structure ID.
+|`MSH` (read only) |Link to received MSH segment data.
+|`Name` (read only) |Message structure name.
 |===
 
 Individual messages have their own maps, with keys matching the segments of the message. For instance, an ACK message would use the message structure ID `ACK`, and the data for the ACK message sent or received would be present as an `ACK` value in the `Data` map. The ACK message is itself a map, and the segments and groups of the message are represented as maps (in the case of singleton instances) or lists of maps (for repeating instances) with positional keys.
```

Comment:
"maps" should be "Map statements", and "map" should be "Map statement" -- please fix globally.